OracleFMWInfrastructure/dockerfiles/12.2.1.4/Dockerfile

```diff
@@ -92,5 +98,6 @@ COPY --from=builder --chown=oracle:oracle /u01 /u01
 RUN chmod +xr $SCRIPT_FILE
 
 USER oracle
+HEALTHCHECK --start-period=30s --interval=30s --timeout=300s  --retries=20 CMD curl -k -s --fail `$HEALTH_SCRIPT_FILE` || exit 1
```

Comment:
These `HEALTHCHECK` values seem far too large. It would take 20x300s to timeout which is 1h20m. I would suggest reducing the `timeout` to ~5-10s and the `retries` to 3-5.